src/index.html

```diff
@@ -1,13 +1,70 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/style.css">
+  </head>
+  <body>
+
+    <div data-qa="card" class="card card--container">
+      <div class="card__photo-container">
+        <img
+          class="card__photo"
+          src="images/imac.jpeg"
+          alt="imac"
+        >
+      </div>
+
+      <div class="card__text">
+        <p class="card__desc">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+      </div>
+  
+      <div class="card__code">
+        <p class="card__product-code">
+          Product code: 195434
+        </p>
+      </div>
+  
+      <div class="card__stars card__stars--reviews">
+        <ul class="card__list">
+          <li class="card__item card__item--one"></li>
+          <li class="card__item card__item--two"></li>
+          <li class="card__item card__item--three"></li>
+          <li class="card__item card__item--four"></li>
+          <li class="card__item card__item--five"></li>
+        </ul>
+
+        <p class="card__review-text">
+          Reviews: 5
+        </p>
+      </div>
+  
+      <div class="card__price card__price--fee">
```

Comment:
Do you really need modifier fee here? Can't you place all the styles just for card__price?